### PR TITLE
Summer BBQ 2018

### PR DESCRIPTION
**Pull request by kz**
To review: [2018-07-25-barbecue.html.pdf](https://github.com/icdocsoc/emails/files/2229260/2018-07-25-barbecue.html.pdf)

### CODE REVIEW

File: xml/2018-07-25-barbecue.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    Hi everyone,
+
+    Hope your summer holidays are going well! With the great weather, DoCSoc
+    will be holding a summer barbecue in a couple weeks' time. If you're around
+    then, take a look at the event details below!
+  </text>
+
+  <hr/>
+
+  <event startDate="2018-08-12T13:00:00+01:00"
```
Comment (nvgrw) [inline xml/2018-07-25-barbecue.xml]: heheee sorry about the ISO 8601

File: xml/2018-07-25-barbecue.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      everyone!
+
+      While there will be lots of food, if you would like to bring some extra
+      food or drinks, that would also be most welcome!
+
+      Note: you must be a member of DoCSoc to attend this event. Computing, JMC,
+      and EIE students are automatically DoCSoc members.
+    </text>
+    <links>
+      <link href="https://www.facebook.com/events/201855943797849/">Facebook</link>
```
Comment (nvgrw) [inline xml/2018-07-25-barbecue.xml]: iff only the fb integration worked 🙈 
how's the progress on that?

File: xml/2018-07-25-barbecue.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    </links>
+  </event>
+
```
Comment (nvgrw) [inline xml/2018-07-25-barbecue.xml]: double newline 👀

File: xml/2018-07-25-barbecue.xml (hunk 2 above)
Comment (kz) [reply xml/2018-07-25-barbecue.xml]: API doesn't seem to be able to retrieve non-subscribed events even with approval. Will have to investigate further.

File: xml/2018-07-25-barbecue.xml (hunk 3 above)
Comment (kz) [reply xml/2018-07-25-barbecue.xml]: Will keep that in mind for next time.

File: xml/2018-07-25-barbecue.xml (hunk 1 above)
Comment (kz) [reply xml/2018-07-25-barbecue.xml]: To be honest, I actually prefer ISO 8601 :P

**Pull request merged**